test/LibraryManager.Test/Providers/JsDelivr/JsDelivrCatalogTest.cs

```diff
@@ -116,10 +112,12 @@ public async Task GetLibraryCompletionSetAsync_Names()
 
         [TestMethod]
         [Ignore] // Enable it after version completion sorting is committed.
+                 // TODO: Also add a test for GitHub version completion
```

Comment:
This test verifies version completion that JsDelivr exposes from NPM packages.  JsDelivr also supports GitHub and provides versions through tags or releases (I can't recall which).  Since *this* test is currently disabled, I didn't want to add another disabled test for the GH scenario, so I added this comment to track it for if/when we implement the requirements here.

Reply:
>JsDelivr also supports GitHub and provides versions through tags or releases (I can't recall which).

Tags, branches and commit hashes but I don't remember if all are implemented here and/or if all also have completion.